crd-generator/api/src/main/java/io/fabric8/crd/generator/visitor/AnnotatedMultiPropertyPathDetector.java

```diff
@@ -84,15 +83,20 @@ public void visit(TypeDefBuilder builder) {
       if (!parents.contains(p) && !excludePropertyProcessing(p)) {
         ClassRef classRef = (ClassRef) p.getTypeRef();
         TypeDef propertyType = Types.typeDefFrom(classRef);
-        if (!propertyType.isEnum()) {
+        if (!propertyType.isEnum() && !classRef.getPackageName().startsWith("java.")) {
```

Comment:
this should be marked as a breaking change, can you add a line to the Changelog?
I'm fine shipping this with 6.10 but it should not be backported to 6.9 imho

Reply:
Can you elaborate on why this would be breaking?  Do you expect users to create their own subclasses / instances that look for annotations that are built-in to java?

Reply:
I expect edge cases where people are relying on `protected` fields inherited from `java.` classes.
Am I missing something?

Reply:
Can you provide an example?  As far as I can see once you reach a property typed as a java. class, there is no scenario where one of its child properties could contain a non-builtin annotation.  If for example you have extended a java class and added your own fields, that would be in your own class - not one from a java. package.  Are you thinking that some users may be modifying java. classes directly?

Reply:
no, your explanation is fair, thanks for providing this detail 👍